exercises/hardware_access_control/README.md

```diff
@@ -35,6 +35,8 @@ This is great!
 If you build and [load][running on fpga] the `hardware_access_part_1` firmware on the FPGA, you have flashing LEDs!
 What more could one want?
 
+*Hint: the UF2 file that you need to copy to `SONATA` is located at: `build/cheriot/cheriot/release/hardware_access_part_1.uf2`*
+
```

Comment:
I feel this may impede the flow. These little reminders could be put in a small section at the beginning of the exercise.

Also, over-use of verbatim, maybe use single quotes for 'SONATA'

Reply:
Ok, I've removed this because it is in the running software guide.